src/test/java/net/minestom/server/entity/EntityMetaTest.java

```diff
@@ -21,4 +30,76 @@ public void ensureRegistration() throws IllegalAccessException {
         }
         assertTrue(list.isEmpty(), "Missing meta for: " + list);
     }
+
+
+    @Test
+    public void notifyAboutChanges(Env env) {
```

Comment:
This should be put in a dedicated `EntityMetaIntegrationTest` file (and allow the changes above to be reverted)

Reply:
Alright, will move it into a new test

Reply:
Should I amend the commit?

Reply:
branch are squashed before merge anyway. so don't worry about the git history

Reply:
ok